scripts/gcs-inventory-loader.ipynb

```diff
@@ -0,0 +1,490 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# GCS Inventory Loader Introduction\n",
+    "Load your GCS bucket inventory into BigQuery (or stdout) fast with this tool.\n",
+    "\n",
+    "It can be very useful to have an inventory of your GCS objects and their metadata, particularly in a powerful database like BigQuery. The GCS listing API supports filtering by prefixes, but more complex queries can't be done via API. Using a database, you can find out lots of information about the data you have in GCS, such as finding very large objects, very old or stale objects, etc.\n",
+    "\n",
+    "This utility will help you bulk load an object listing to stdout, or directly into BigQuery. It can also help you keep your inventory up-to-date with the listen command.\n",
+    "\n",
```

Comment:
The introduction is great! However, if I had never used it before I think it is a tiny bit confusing what is actually happening in the notebook so maybe a summary section like:
1. Set up the Notebook
2. Load your workspace bucket to a BQ project
3. Visualize object metadata


Also maybe a note or two about how they have to come to this notebook with a BQ project and dataset already created.